custom-namespaces.yaml

```diff
@@ -423,26 +390,11 @@ firefox_accounts:
         - channel: release
           table: mozdata.firefox_accounts.fxa_log_device_command_events
   explores:
-    all_event_counts:
-      type: events_explore
-      views:
-        base_view: all_events
-        extended_view: all_events_table
```

Comment:
For some reason the data catalog doesn't show the `firefox_accounts.all_event_counts` explore, so I'm not sure how to tell what all might be using that explore, but I do know @ksiegler1 was planning to build some Looker dashboards using it.

@ksiegler1 have you created any dashboards/charts using the `firefox_accounts.all_event_counts` explore yet?